docs/manual_api/cpp_tile_edsl.rst

```diff
@@ -0,0 +1,1070 @@
+C++ Tile eDSL
```

Comment:
I don't think this file should go into `docs/manual_api`, perhaps into `docs/articles` or `docs/usage`.

Reply:
Additionally, this file should probably be renamed to `tile_edsl.rst` and any content that's not specific to a particular language should have language-specific references removed. Then we can start adding tabs for language-specific examples, and have a more generalized document overall. Thoughts?

Reply:
Done